### PR TITLE
Add dot-org slack  worfklow template

### PR DESCRIPTION
**Pull request by rasquith**
- Add a workflow to notify a slack channel about messages bug reports with the anaconda.org checkbox selected

### CODE REVIEW

File: .github/workflows/dot-org-filter.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        fi
+
+  notify-slack:
+    if: needs.check-conditions.outputs.should-notify == 'true'
+    runs-on:
+      labels: self-hosted-main
+    needs: [check-conditions]
+    steps:
+    - name: Send message to Slack
+      uses: rtCamp/action-slack-notify@v2
```
Comment (rasquith) [inline .github/workflows/dot-org-filter.yml:34]: this is an external extension, so I'm looking for what we typically use. I noticed some more complex flows with bots.

File: .github/workflows/dot-org-filter.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+on:
+  issues:
+    types: [opened, edited, labeled]
+
+jobs:
+  check-conditions:
+    if: contains(github.event.issue.labels.*.name, 'type-bug')
+    runs-on:
+      labels: self-hosted-main
```
Comment (pww217) [inline .github/workflows/dot-org-filter.yml:11]: Not available in this org. Probably `ubuntu-latest` is fine.

File: .github/workflows/dot-org-filter.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+name: Anaconda.org Isuse Notification to Slack
```
Comment (pww217) [inline .github/workflows/dot-org-filter.yml:1]:
```suggestion
name: Anaconda.org Issue Notification to Slack
```

Comment (pww217) [reply .github/workflows/dot-org-filter.yml:1]: Also often good to add a couple lines of comments under this describing the workflow's purpose at a high level

**Review by elize-anaconda — APPROVED** [review]

**Pull request merged**